app/lib/methods/helpers/openLink.ts

```diff
@@ -38,6 +39,16 @@ const appSchemeURL = (url: string, browser: string): string => {
 };
 
 const openLink = async (url: string, theme: TSupportedThemes = 'light'): Promise<void> => {
+	const telRegExp = new RegExp(/^(tel:)/)
+	if (telRegExp.test(url)) {
+		try {
+			await Linking.openURL(url);
+			return
+		} catch (e) {
+			log(e)
```

Comment:
Missing semicolons. Saw in some other places as well.
Make sure you configure your vscode correctly to run Prettier, man.

Reply:
Yeah, you are right! Sorry! FIxed now 👍